src/ProfiledAot/src/CommonMethods.cs

```diff
@@ -1,19 +1,27 @@
 // This class represents common code paths we always want to AOT
 
+// string interpolation
+// Culture-aware string comparisons
 // System.Threading.Tasks.Task
 // System.Net.Http.HttpClient
```

Comment:
Were you going to also add MS.Extensions.Http?  or is that in a future change?

Reply:
After testing some more, I don't find any `AOT NOT FOUND` messages for this in the Podcast app.

Every line with "http" is:
````
03-09 14:58:41.368 11672 11672 D Mono    : AOT NOT FOUND: (wrapper managed-to-managed) Microsoft.Extensions.Http.IHttpMessageHandlerBuilderFilter[]:System.Collections.Generic.ICollection`1.get_Count ().
03-09 14:58:41.516 11672 11672 D Mono    : AOT NOT FOUND: Microsoft.Extensions.Logging.Logger:BeginScope<Microsoft.Extensions.Logging.LoggerMessage/LogValues`2<System.Net.Http.HttpMethod, System.Uri>> (Microsoft.Extensions.Logging.LoggerMessageLogValues`2<System.Net.Http.HttpMethod, System.Uri>).
03-09 14:58:41.530 11672 11709 D Mono    : AOT NOT FOUND: System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1/AsyncStateMachineBox`1<Java.Net.HttpURLConnection, Xamarin.Android.Net.AndroidMessageHandler/<SetupRequestInternal>d__146>:ExecuteFromThreadPool (System.Threading.Thread).
03-09 14:58:41.531 11672 11672 D Mono    : AOT NOT FOUND: System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1<System.Collections.Generic.IEnumerable`1<Microsoft.NetConf2021.Maui.Models.Responses.ShowResponse>>:AwaitUnsafeOnCompleted<System.Runtime.CompilerServices.TaskAwaiter`1<System.Net.Http.HttpResponseMessage>, Microsoft.NetConf2021.Maui.Services.ShowsService/<TryGetAsync>d__9`1<System.Collections.Generic.IEnumerable`1<Microsoft.NetConf2021.Maui.Models.Responses.ShowResponse>>> (System.Runtime.CompilerServices.TaskAwaiter`1<System.Net.Http.HttpResponseMessage>&,Microsoft.NetConf2021.Maui.Services.ShowsService/<TryGetAsync>d__9`1<System.Collections.Generic.IEnumerable`1<Microsoft.NetConf2021.Maui.Models.Responses.ShowResponse>>&).
03-09 14:58:41.533 11672 11672 D Mono    : AOT NOT FOUND: System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1<System.Collections.Generic.IEnumerable`1<Microsoft.NetConf2021.Maui.Models.Responses.ShowResponse>>:AwaitUnsafeOnCompleted<System.Runtime.CompilerServices.TaskAwaiter`1<System.Net.Http.HttpResponseMessage>> (System.Runtime.CompilerServices.TaskAwaiter`1<System.Net.Http.HttpResponseMessage>&,System.Runtime.CompilerServices.IAsyncStateMachineBox).
````